packages/internal/src/debug.js

```diff
@@ -17,7 +17,7 @@ export const makeTracer = (name, enable = true) => {
     }
     case 'verbose': {
       const infoTick = (optLog, ...args) => {
-        if (optLog.log) {
+        if (typeof optLog?.log === 'function') {
```

Comment:
This also narrows the sniffing to `optLog.log` being a function. Previously, it could be any object with a `log` property whose value was truthy. Nevertheless, this kind of sniffing is a bad smell, and will misbehave on an object that accidentally passes this sniff test.